src/components/common/Graphs/LineGraph.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import axios from "axios";
+import Plot from "react-plotly.js";
+import LoadingComponent from "../../common/LoadingComponent";
+
+export default function LineGraph({ state, state2, state3 }) {
+  const [data, setData] = React.useState(null);
+  const [data2, setData2] = React.useState(null);
+  const [data3, setData3] = React.useState(null);
+
+  React.useEffect(() => {
+    axios
+      .get(`https://b-ds.citrics.dev/viz/${state}`)
+      .then(({ data }) => setData(JSON.parse(data).data));
+    if (state2) {
+      axios
+        .get(`https://b-ds.citrics.dev/viz/${state2}`)
+        .then(({ data }) => setData2(JSON.parse(data).data));
+    }
+    if (state3) {
+      axios
+        .get(`https://b-ds.citrics.dev/viz/${state3}`)
+        .then(({ data }) => setData3(JSON.parse(data).data));
+    }
+  }, [state, state2, state3]);
+
```

Comment:
I wonder if this might make more sense as three separate useEffect-s.

Reply:
This is just for testing purposes. Ideally we'll have all the data when the cities are selected so we won't need to do any axios requests.

Reply:
Ahh gotcha! Definitely not worth optimizing now then.